pkg/forwarder/forwarder.go

```diff
@@ -373,6 +393,10 @@ func (f *DefaultForwarder) sendHTTPTransactions(transactions []*HTTPTransaction)
 	}
 
 	for _, t := range transactions {
+		payloadInputs.Add(t.Endpoint, 1)
+		payloadInputSizes.Add(t.Endpoint, int64(len(*t.Payload)))
+		tlmPayloadInputs.Add(1, t.Endpoint)
+		tlmPayloadInputSizes.Add(float64(len(*t.Payload)), t.Endpoint)
```

Comment:
Just my guess, someone else probably knows better, but I think because some metrics like histograms and distributions might require the additional precision.